### DISCUSSION TITLE
installation error Tandoor Recipes is not yet available

### DISCUSSION THREAD

**Discussion by Chrysanthemum536** [TandoorRecipes/recipes#3203] [Q&A] [answered]
installing with docker compose and got this error:
```
Tandoor Recipes is not yet available 502

Services are still trying to start.
Please allow up to 3 minutes after you started the application on your server.

If this status persists, check the application or docker logs for further information.
After checking and trying everything mentioned in the docs, you can request help on the project's GitHub page.
```

and it did not start after 3 mintues.

in the logs for web:
```
/opt/recipes/venv/bin/gunicorn: line 3: import: not found

/opt/recipes/venv/bin/gunicorn: line 4: import: not found

/opt/recipes/venv/bin/gunicorn: line 5: from: not found

/opt/recipes/venv/bin/gunicorn: line 7: syntax error: unexpected "(" (expecting "then")
```

**Comment by smilerz** [comment]
Please share the entire log.

**Reply by Chrysanthemum536** [reply]
[_tandoor-web_recipes-1_logs.txt](https://github.com/user-attachments/files/16076605/_tandoor-web_recipes-1_logs.txt)

**Reply by Chrysanthemum536** [reply]
[docker-compose.yml.txt](https://github.com/user-attachments/files/16076618/docker-compose.yml.txt)

**Reply by smilerz** [reply]
Is that after a fresh start?

**Reply by Chrysanthemum536** [reply]
yes it still happens after deleting the containers and volumes

**Reply by smilerz** [reply]
delete the container, volumes and build again using `docker-compose build --no-cache`

**Reply by Chrysanthemum536** [reply]
I got the output 
```
docker-compose build --no-cache
db_recipes uses an image, skipping
web_recipes uses an image, skipping
nginx_recipes uses an image, skipping
```
and it still has the same error after running docker-compose up -d

**Reply by smilerz** [reply, marked as answer]
oops, wrong command

`docker-compose pull && docker-compose -f docker-compose.yml up -d`

**Reply by Chrysanthemum536** [reply]
Thanks, it is working now
